cl/assets/static-global/js/base.js

```diff
@@ -176,6 +176,28 @@ $(document).ready(function () {
     }
   });
 
+  ///////////////////////////
+  // TOC Collapse Controls //
+  ///////////////////////////
+
+  $('.collapse').on('shown.bs.collapse', function () {
+    // do something…
```

Comment:
Can you improve these comments, please, and then I think we're good to go.

Reply:
Oh, yes, sorry! That's the placeholder from the bootstrap example I copied.